frontend/packages/console-app/src/components/quick-starts/catalog/QuickStartCatalogFilter.scss

```diff
@@ -0,0 +1,18 @@
+.co-quick-start-catalog-filter {
+  &__flex {
+    padding: var(--pf-global--spacer--sm) var(--pf-global--spacer--lg);
+    border-top: 1px solid var(--pf-global--BorderColor--100);
+  }
+
+  &__input {
+    width: 500px;
+  }
+
+  &__status {
+    margin-left: 5px;
```

Comment:
pf variables should be used for constants.

Reply:
Won't be needing `__status` anymore since dropdown is replaced with `Select`